Common++/header/IpAddress.h

```diff
@@ -81,7 +87,16 @@ namespace pcpp
 		/**
 		 * Overload of the less-than operator
 		 */
-		bool operator<(const IPv4Address& rhs) const { return toInt() < rhs.toInt(); }
+		bool operator<(const IPv4Address& rhs) const
+		{
+			uint32_t intVal = toInt();
+			std::reverse((uint8_t*)(&intVal), (uint8_t*)(&intVal) + sizeof(intVal));
+
+			uint32_t rhsIntVal = rhs.toInt();
+			std::reverse((uint8_t*)(&rhsIntVal), (uint8_t*)(&rhsIntVal) + sizeof(rhsIntVal));
+
+			return intVal < rhsIntVal;
+		}
```

Comment:
Can you please update your branch from `PcapPlusPlus/dev`? This change is already merged...

Reply:
Done